sass/variables/_sizing.scss

```diff
@@ -160,3 +161,7 @@ $table-cell-padding-top-and-bottom: $base-padding;
 
 $table-data-icon-size: 2em;
 $table-border-spacing: 7.5px;
+
+// Accordions
+$accordion-bottom-margin: $form-fieldset-margin;
```

Comment:
@toastercup Good thinking to add a variable, but I'm changing the name and value of this variable. 
* In every instance, the variable replaces a calculated value for border width, so renaming it to `$accordion-bottom-border`
* The value of that border-width was previously 2px and there's nothing in the story to indicate that should have changed, so I'm using the previous calculation as the value of the variable
* `$form-fieldset-margin` is used on the `fieldset` tag and isn't related to any of the elements in the FAQs: https://github.com/cb-talent-development/employer-style-base/blob/topic/EM-1322-Subscription-FAQ/sass/_forms.scss#L115